.github/workflows/buid_and_release.yml

```diff
@@ -40,4 +40,6 @@ jobs:
         prerelease: false
         files: |
           omim.ttl
```

Comment:
```suggestion
```

cc @joeflack4 as per your comment below about "why both TTL and OWL".

Reply:
Thanks, yes I think [as we discussed](https://github.com/monarch-initiative/omim/pull/107#discussion_r1477474080), this is a good idea.

I went ahead and made the necessary changes in `mondo-ingest` to account for this. Can you merge that after merging this PR?:
- https://github.com/monarch-initiative/mondo-ingest/pull/433